doc/classes/GraphEdit.xml

```diff
@@ -375,12 +375,13 @@
 		<signal name="node_deselected">
 			<param index="0" name="node" type="Node" />
 			<description>
+				Emitted when the GraphNode [param node] is deselected.
 			</description>
 		</signal>
 		<signal name="node_selected">
 			<param index="0" name="node" type="Node" />
 			<description>
-				Emitted when a GraphNode is selected.
+				Emitted when the GraphNode [param node] is selected.
```

Comment:
Both of these should reference `GraphElement`, not `GraphNode`. Same is true for `begin_node_move`/`end_node_move`.

`copy_nodes_request`, `paste_nodes_request`, and `duplicate_nodes_request` don't have any type checks and are triggered on input actions, so their descriptions should also refer to `GraphElement`, I think. (Also `duplicate_nodes_request`'s description is awkwardly phrased).

`delete_nodes_request` is also triggered by an action, but it explicitly passes a collection of nodes for some reason, and those indeed are `GraphNode` and not `GraphElement`. Not sure if this makes sense (and this is something we can still break compatibility for). But for now the description makes sense there, but it should be reworded, as it refers to a single node and to having a closing button.

Reply:
All connection related signals are `GraphNode` specific, so those are probably all fine.

Reply:
Always amusing to notice poorly worded descriptions that are so poorly worded they're actually misleading.
I should only adjust a few without going out of scope for the PR, probably.

Reply:
I think adjusting descriptions of those would be in scope (you can basically rely on the same template for all of them). But doing any changes to the graph nodes themselves to address inconsistencies is of course out of scope, just me musing.